ebos/eclproblem.hh

```diff
@@ -1,6 +1,8 @@
 // -*- mode: C++; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*-
 // vi: set et ts=4 sw=4 sts=4:
 /*
+  Copyright 2023 INRIA
```

Comment:
If you add one copyright holder then you should add the others, too. Otherwise this might leave a wrong impression.
Of course this is kind of hard to do, but we should try.
As inspiration: For the Debian packages this  (and other) file's copyright is documented as:
```
Copyright: 2009-2022 OPM development team
Comment: We use the name "OPM development team", to denote that parts
     of individual files might be copyright by
     Dr. Blatt - HPC-Simulation-Software & Services, Equinor ASA,
     Statoil ASA (renamed to Equinor ASA in 2018),
     IWS-LH2 at University of Stuttgart, NORCE,
     IRIS (merged into NORCE in 2018), OPM-OP AS, Poware (Andreas Lauser),
     SINTEF, SIMULA, or TNO.
```

Reply:
I think that having a separate list of contributors and a generic header seems like a good idea. Then I don't have to remember to add INRIA to the ones I contribute to.